docs/community/code_of_conduct.md

```diff
@@ -72,9 +70,8 @@ I required a mailing list, a voice chat and text chat on discord and access to t
 The officers will evaluate your request and either reject or reach out with further instruction and access keys.
 
 
-## Credit:
+## **Credit:**
 
-Whenever someone else's work is used it needs to be given credit and as much as possible referenced.  Ie.  Linking to graphic designer website if said person’s artwork was used. 
+Whenever someone else's work is used it needs to be given credit and as much as possible referenced.  Ie.  Linking to the graphic designer’s website if said person’s artwork was used. 
```

Comment:
I think that it would be better written as.

Whenever someone else's work is used it needs to be given credit and referenced as much as possible. Ie. Linking to the graphic designer’s website if said person’s artwork was used.

Reply:
👍 will do.